src/pages/resultPage.js

```diff
@@ -1,11 +1,22 @@
-import { USER_INTERFACE_ID } from '../constants.js';
+import { RETRY_BUTTON_ID, USER_INTERFACE_ID } from '../constants.js';
 import { createResultComponent } from '../components/resultComponent.js';
+import { quizData } from '../data.js';
+import { initQuestionPage } from './questionPage.js';
 
 export const initResultPage = () => {
   const element = document.getElementById(USER_INTERFACE_ID);
   element.innerHTML = ``;
   const resultElement = createResultComponent();
   element.appendChild(resultElement);
 
+  document
```

Comment:
Please remove this and move the event handling to the `resultComponent` and pass an argument `onRetry` to `createResultComponent`